src/containers/City.js

```diff
@@ -1,14 +1,13 @@
 import React, { useEffect, useState } from 'react';
 import { useParams } from 'react-router-dom';
-
+import { useBusinesses } from '../helpers/functions';
 import { CategorySelector, DiscoveryPage } from '../components';
 import { toDisplayName } from '../helpers/utils';
-import { useBusinessesInStateAndCity } from '../helpers/db';
 
 const City = () => {
   const { state, city } = useParams();
   const [categories, setCategories] = useState([]);
-  const { businesses, loading } = useBusinessesInStateAndCity(state, city);
+  const { businesses, loading } = useBusinesses(state, city, undefined);
```

Comment:
Can we make the second and third params optional instead of passing in undefined?

Reply:
Yeah, I can probably just not pass them and it should be okay. I just passed undefined to be more clear about things. At least in my eyes.

Reply:
I generally like keeping the undefined to keep a consistent "look" to the js functions. JS does not really have optional parameters in the same was as c#. If you want me to remove this and change to a different paradigm I can. But it would not work well if we had to leave out a middle param.

Reply:
I see. We generally try to avoid passing in nulls because you have to assume what null means in that instance. Is there a situation where you pass in state and category but not city?

This also isn't a huge deal, so we can leave as is if we want

Reply:
There could be a time where we get all the restaurants in Nebraska, perhaps. I will look into a better way to do this that I am not aware of after I get moved into Kuaff this morning.